### PR TITLE
Remove broken connect button

### PR DESCRIPTION
**Pull request by clarkkozak**
removed BlankConnectModal to prevent confusion
built project to have latest build
fixed type errors to cut down on the large build log

### CODE REVIEW

File: dist/widgets/Menu/config.d.ts [hunk 1]
```diff
@@ -23,11 +23,7 @@ export declare const links: ({
     calloutClass: string;
     href?: undefined;
 })[];
-export declare const socials: {
-    label: string;
-    icon: string;
-    href: string;
-}[];
+export declare const socials: any;
```
Comment (clarkkozak) [inline dist/widgets/Menu/config.d.ts:26]: explicit type any to remove type warnings during compilation

File: dist/widgets/WalletModal/useWalletModal.d.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import { Login } from "./types";
 interface ReturnType {
     onPresentConnectModal: () => void;
-    onPresentConnectModal1: () => void;
```
Comment (clarkkozak) [inline dist/widgets/WalletModal/useWalletModal.d.ts:4]: This was most likely used while building the project and is no longer needed. Removed from types as well

File: src/widgets/Menu/Menu.tsx [hunk 3]
```diff
@@ -130,10 +130,6 @@ const Menu: React.FC<NavProps> = ({
           mobileLightLogoUrl={mobileLightLogoUrl}
           mobileDarkLogoUrl={mobileDarkLogoUrl}
         />
-        <Flex>
-          <UserBlock account={account} login={login} logout={logout} />
-          {profile && <Avatar profile={profile} />}
-        </Flex>
```
Comment (clarkkozak) [inline src/widgets/Menu/Menu.tsx:136]: ⭐ Removed broken connect button

File: src/widgets/Menu/components/SocialLinks.tsx [hunk 4]
```diff
@@ -10,14 +10,14 @@ const Icons = (IconModule as unknown) as { [key: string]: React.FC<SvgProps> };
...
-    {socials.map((social, index) => {
+    {socials.map((social: any, index: number) => {
       const Icon = Icons[social.icon];
       const iconProps = { width: "24px", color: "textSubtle", style: { cursor: "pointer" } };
       const mr = index < socials.length - 1 ? "24px" : 0;
       if (social.items) {
         return (
           <Dropdown key={social.label} position="top" target={<Icon {...iconProps} mr={mr} />}>
-            {social.items.map((item) => (
+            {social.items.map((item: any) => (
```
Comment (cryptoquick) [inline src/widgets/Menu/components/SocialLinks.tsx:20]: I don't think you should be using any here

File: src/widgets/Menu/config.ts [hunk 5]
```diff
@@ -113,7 +113,7 @@ export const links = [
   },
 ];
 
-export const socials = [
+export const socials: any = [
```
Comment (cryptoquick) [inline src/widgets/Menu/config.ts:116]: no any

**Review by cryptoquick — CHANGES_REQUESTED** [review]
Why all the any's?

**Comment by clarkkozak** [comment]
Fixed explicit any and removed the warning 💪🏼

**Review by cryptoquick — APPROVED** [review]
Awesome, LGTM!

**Pull request merged**